var/spack/repos/builtin/packages/py-tensorflow-probability/package.py

```diff
@@ -60,8 +60,8 @@ class PyTensorflowProbability(Package):
 
     variant("tf", default=False, description="Build with TensorFlow support")
     with when("+tf"):
-        depends_on("py-tf-keras@2.16:", when="@0.24:", type=("build", "run"))
-        depends_on("py-tensorflow@2.16", when="@0.24", type=("build", "run"))
+        depends_on("py-tf-keras@2.17:", when="@0.24:", type=("build", "run"))
+        depends_on("py-tensorflow@2.17", when="@0.24", type=("build", "run"))
```

Comment:
This is no longer correct, we can't simply change the numbers to make it concretize, we need to add new versions of packages.